src/disputables/index.js

```diff
@@ -229,27 +234,38 @@ function buildDisputedActionRadspec(transactions) {
  * @param {String} appAddress Address of the app
  * @param {String} entityPath Entity where the action is being disputed (e.g vote, delay, etc)
  * @param {String} actionId Disputable action id
+ * @param {String} arbitrableAddress Address of the arbitrable
  * @returns {String} URL of the disputed action.
  */
 function buildDisputedActionUrl(
   organization,
   appAddress,
   entityPath,
-  actionId
+  actionId,
+  arbitrableAddress
 ) {
-  const orgUrl = buildOrgUrl(organization)
-  return [orgUrl, appAddress, entityPath, actionId].join('/')
+  const networkType = sanitizeNetworkType(getNetworkType())
+  const arbitrable = getKnownArbitrable(networkType, arbitrableAddress)
+
+  let url
+  if (arbitrable) {
+    url = [arbitrable.link, arbitrable.entityPath, actionId].join('/')
```

Comment:
Not a big deal, but maybe this is something we would hide inside the API since each arbitrable could have its own URL scheme?

Perhaps something like `isArbitrableKnown(arbitrableAddress, networkType) && buildArbitrableUrl(arbitrableAddress, actionId, networkType)` would be better? And then the mapping would be from `address => function`.